docs/lib/datastore/fragments/ios/auth-model.md

```diff
@@ -0,0 +1,104 @@
+<amplify-callout>
+When using the `@auth` directive with DataStore, currently only following fields are supported:
+
+1. `allow` with "owner" and "groups"
+2. `ownerField`
+3. `groups`
+4. `operations`
+
+</amplify-callout>
+
```

Comment:
I don't think this is deserving of a call out.  If we start to use them too much, people will stop reading them.

We could have a lead in paragraph just saying something like...
The @auth directive when used with Amplify DataStore supports the following fields:
- a 
- b
- c 
- d